apps/acrs/translations/src/en/validation.json

```diff
@@ -236,7 +236,14 @@
     "has-family-member-been-evacuated": {
       "required": "You must select an option"
     },
-    "how-to-send-decision": {
+    "image": {
+      "default": "We have not been able to upload evidence",
+      "unsaved": "We have not been able to upload evidence",
+      "maxFileSizeInBytes": "Select a file that is smaller than 25 MB",
+      "fileType": "This file is not a valid file format you can upload to this service.",
+      "tooMany": "You can only submit up to 250 attachments."
+    },
+    "how-send-decision": {
```

Comment:
this has been changed from `how-to-send-decision` to `how-send-decision`, however the field name in fields/index.js is still remains the same - `how-to-send-decision`

Reply:
Changed this back to `how-to-send-decision`.